tests/serverpod_test_server/test/connection_test.dart

```diff
@@ -74,6 +76,22 @@ void main() {
       expect(result[2], equals(2));
     });
 
+    test('List<List<int>> parameter and return type', () async {
```

Comment:
Should tests like this also be added for List<List<String>>, List<List<String>>?, List<List<double>>, ...?